app/controllers/reviews_controller.rb

```diff
@@ -29,12 +29,18 @@ def edit
   def update
     @spot = Spot.find params[:spot_id]
     @review = Review.find(params[:id])
-    if @review.update(review_params)
-      flash[:success] = "Review successfully updated"
-      redirect_to spot_path(@spot)
+    if (current_user && current_user.id == @review.user_id) ||
+      (current_user && current_user.admin?)
```

Comment:
Use 4 (not 2) spaces for indenting a condition in an `if` statement spanning multiple lines.